lte/gateway/c/core/oai/include/mme_init.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2022 The Magma Authors.
```

Comment:
What's the purpose of creating this new file? 

Further, this is initializing the S1AP task, not the MME task. So the name is misleading.

Reply:
1. All tasks are initialized from oai_mme.c file in main function. 
2. As we port the individual tasks to cpp, initialize functions are called from main function. And all these initialize shall be put in this file.
3. Since we started with s1ap task, right now only s1ap task's functionalities are available

Reply:
Can you add this to the PR description and also the parent issue to guide the migration strategy ? Also, is the goal to eventually get rid of this file?